angular/core/components/dashboard_page/dashboard_page.scss

```diff
@@ -55,3 +57,14 @@
 .dashboard-page__show-more a:hover{
   color: $primary-text-color;
 }
+
+.dashboard-page__mute-image--large {
+  img { max-width: 730px; }
```

Comment:
That was the width of the text (I think due to the single column layout class). I've updated it now though :)